### ISSUE TITLE
Templating isn't working

### ISSUE THREAD

**Issue by Slemener** [stevenschobert/instafeed.js#291]
Hello,

I tried to use the templating feature with instafeed but once I add the template line into the code, nothing shows up. The #instafeed div is totally empty. When I don't try to use template, it's perfectly working.

Here is my code :

    <script type="text/javascript">
    var feed = new Instafeed({
        get: 'user',
		userId: MY_APP_ID,
		 clientId: '439e58a30f834d1eb7056cda2bf48536',
        accessToken: 'MY_ACCESS_TOKEN'
        template: '<a target="_blank" href="{{link}}"><img src="{{image}}" /></a>'
    });
    feed.run();
    </script>

Of course, I use the real values for userId and accessToken (just didn't know if it's safe to write it ???)

The console says : Uncaught SyntaxError : Unexpected identifier at line 60 (the template line !!!)

Unfortunately, I can't show you a live example. But as I said, it's only when I try to use template with get: 'user' that I have problems. It worked with get: 'tagged'.

Can you help me ? I don't understand what's wrong.

Sorry for my English I'm not a native speaker.

Thanks a lot :)

**Comment by stevenschobert** [comment]
@Slemener it looks like you might be missing a comma at the end of your `accessToken: 'MY_ACCESS_TOKEN'` line, was that the issue?

**Comment by Slemener** [comment]
Thank you Steven! And sorry for the time I took to answer. Indeed, it was the missing coma :) 
I have some other question to ask : Can I use Instafeed.js for a professional / commercial use ? I was also wondering about the security issue when you use the client-side (Implicit) Authentication to get the token. Is it really secure, if I just embed my Instagram feed on a webpage ? I'm not sure I understood how to set the token to basic authorization only.

Thank you for your help. You did a great job with Instafeed!
